src/Servers/IIS/IIS/test/Common.FunctionalTests/PublishedSitesFixture.cs

```diff
@@ -19,13 +19,11 @@ public class PublishedSitesCollection : ICollectionFixture<PublishedSitesFixture
 
     public class PublishedSitesFixture : IDisposable
     {
-        public CachingApplicationPublisher InProcessTestSite { get; } = new CachingApplicationPublisher(Helpers.GetInProcessTestSitesPath());
-        public CachingApplicationPublisher OutOfProcessTestSite { get; } = new CachingApplicationPublisher(Helpers.GetOutOfProcessTestSitesPath());
+        public PublishedApplicationPublisher InProcessTestSite { get; } = new PublishedApplicationPublisher(Helpers.GetInProcessTestSitesPath());
+        public PublishedApplicationPublisher OutOfProcessTestSite { get; } = new PublishedApplicationPublisher(Helpers.GetOutOfProcessTestSitesPath());
 
         public void Dispose()
         {
-            InProcessTestSite.Dispose();
```

Comment:
Why is this no longer needed?

Reply:
We were publishing and catching published output before. This was a way to clean up the cache which is no longer required.